packages/pigeon/platform_tests/shared_test_plugin_code/lib/integration_tests.dart

```diff
@@ -246,6 +325,11 @@ class _FlutterApiTestImplementation implements FlutterIntegrationCoreApi {
     return everything;
   }
 
+  @override
+  AllNullableTypes echoAllNullableTypes(AllNullableTypes everything) {
+    return everything;
+  }
+
```

Comment:
It looks like you forgot to actually add most of the new tests :) All of the new `echoNullable*` methods should have two tests each, once calling with a value, and once calling with null.

Reply:
sigh, that'll be in another pr. This one got large enough without adding another 8-10 tests per language

Reply:
I'm back on my phone, but didn't you already add all the native implementations in this PR and it's just the Dart tests to call them that's missing? That's what I thought when I left this comment.

Reply:
Oh, to clarify: I just meant adding integration tests, not native unit tests.

Reply:
nah, you're right, just need the dart logic.

Reply:
I'll add it to this pr later today